google/pubsub_v1/services/publisher/client.py

```diff
@@ -26,6 +26,7 @@
 from google.api_core import exceptions  # type: ignore
 from google.api_core import gapic_v1  # type: ignore
 from google.api_core import retry as retries  # type: ignore
+from google.api_core import timeout as timeouts  # type: ignore
```

Comment:
This is a generated file, but I included it in this PR so that it can be seen what the synth changes will do to it.